tests/test_datasets/test_loading.py

```diff
@@ -63,10 +63,15 @@ class MockTarFileSingleDataset(TarFileSingleDataset):
     """Mock downloading a tar.gz archive with a single file."""
 
     def __init__(self, cache_root: str):
-        super().__init__(url=..., name=..., relative_path='nations/train.txt', cache_root=cache_root)
```

Comment:
maybe we should have two tests here to make sure it works both ways

Reply:
nevermind, since all strings get processed into path objects internally, the tests with strings hit both.